ansible-cli.js

```diff
@@ -1,197 +1,93 @@
 const { promisify } = require("util");
 const childProcess = require("child_process");
 const _ = require("lodash");
-const {
-  validatePaths, prependNamedArguments,
-} = require("./helpers");
 const { ANSIBLE_DOCKER_IMAGE } = require("./consts.json");
 const {
   createDockerVolumeConfig,
-  createDockerVolumesString,
-  createEnvironmentVariablesString,
+  createDockerVolumeArguments,
+  createEnvironmentVariableArguments,
 } = require("./docker-helpers");
+const { logToActivityLog } = require("./helpers");
 
 const exec = promisify(childProcess.exec);
 
 async function execute({
-  params: ansibleParams,
+  params,
+  additionalArguments,
   command,
 }) {
-  const volumeConfigsMap = await createDockerVolumeConfigsMap(ansibleParams);
+  const volumeConfigsMap = createVolumeConfigsMap(params);
   const volumeConfigsArray = [...volumeConfigsMap.values()].flat();
   const environmentVariables = volumeConfigsArray.reduce((acc, curr) => ({
     ...acc,
     ...curr.environmentVariables,
   }), {});
-  const ansibleCommandParams = createAnsibleCommandParams(ansibleParams, volumeConfigsMap);
 
-  const ansibleCommand = createAnsibleCommand(command, ansibleCommandParams);
+  const updatedParams = params;
+  if (volumeConfigsMap.has("vaultPasswordFile")) {
+    updatedParams.vaultPasswordFile = `$${volumeConfigsMap.get("vaultPasswordFile").mountPoint}`;
```

Comment:
Since `updatedParams` and `params` share the reference, this line will update both of them. Clone the object, to avoid source object mutation.